cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -43,8 +43,18 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
       text = "When creating a unique constraint, a name can be provided. The constraint ensures that your database " +
         "will never contain more than one node with a specific label and one property value.",
       queryText = "CREATE CONSTRAINT constraint_name ON (book:Book) ASSERT book.isbn IS UNIQUE",
-      optionalResultExplanation = "",
-      assertions = (p) => assertConstraintWithNameExists("constraint_name", "Book", List("isbn"))
+      assertions = _ => assertConstraintWithNameExists("constraint_name", "Book", List("isbn"))
+    )
+    prepareAndTestQuery(
+      title = "Create a unique constraint only if it does not already exist",
+      text = "If it is unknown if a constraint exists or not but we want to make sure it does, we add the `IF NOT EXISTS`. " +
+        "Note: The `IF NOT EXISTS` syntax for constraints is only available in Neo4j 4.1.3 and onwards. " +
```

Comment:
This is not rendering as a note in the way I think you wanted.... 
![Screenshot 2020-10-01 at 19 31 09](https://user-images.githubusercontent.com/12032446/94849027-b65bd880-041c-11eb-9a26-2d960aff6425.png)

Reply:
We have not figured out how to do notes in the way we want in this version of the setup, we have 3 different ways where 2 are different test set-ups. Only one of the test set-up versions have a clear way of making notes, the other is just inline in the text. (You could compare with the https://github.com/neo4j/neo4j-documentation/pull/654 which has notes using all three set-ups)

My comment on these when looking at the build was to put the note in the end of the section instead of the middle to make it a little better.